regression/goto-cl/Fo/Fo-directory.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+
+/c main1.c main2.c /Fo dir
```

Comment:
I believe the `clean` target of the `Makefile` needs to be expanded for this one, adding `*.obj`.